docs/robots.txt

```diff
@@ -0,0 +1 @@
+Sitemap: https://chaoss.github.io/open-source-in-asia/sitemap.xml
```

Comment:
Would be this the right sitemap since the site will essentially point to: https://chaoss.github.io/oscdb?